jujugui/static/gui/src/app/app.js

```diff
@@ -993,8 +983,34 @@ YUI.add('juju-gui', function(Y) {
           // If the charmbrowser is open then don't show the logout link.
           visible={!this.state.current.store}
           locationAssign={window.location.assign.bind(window.location)}
-        />,
-        linkContainer);
+        />);
+
+      const navigateUserProfile = () => {
+        const auth = this._getAuth();
+        if (!auth) {
+          return;
+        }
+        views.utils.showProfile(
+         this.env && this.env.get('ecs'),
+         this.state.changeState.bind(this.state), auth.rootUserName);
```

Comment:
can you move the `auth.rootUserName` to a new line? I kept reading this as part of the `changeState` call :D